src/Microsoft.Azure.WebJobs.Host/Singleton/LockHandle.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.WebJobs.Host.Timers;
+
+namespace Microsoft.Azure.WebJobs.Host
+{
+    /// <summary>
+    /// Internal lock handle returned by <see cref="SingletonManager"/>
```

Comment:
I'll rename. It's internal. 

---
In reply to: [126262728](https://github.com/Azure/azure-webjobs-sdk/pull/1202#discussion_r126262728) [](ancestors = 126262728)